css/style.css

```diff
@@ -0,0 +1,830 @@
+  padding: 0;
+}
+
+.distance__text {
+  display: flex;
+  width: 29px;
+  height: 25px;
+  margin: 15px 1px 0;
+  padding: 0;
+}
+
+.distance__subtext {
+  display: flex;
+  width: 67px;
+  height: 15px;
+  font-style: normal;
+  font-weight: normal;
+  text-align: left;
+  margin: -18px 0 0 40px;
+  padding: 0;
+}
+
+.test-drive {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+/* руль тест */
+
+.test-drive__rudder {
+  display: flex;
+  width: 22px;
+  height: 58px;
+  margin: 0 6px 0 0;
+  background: url(https://i.ibb.co/wWgHDvw/steering-wheel.png) no-repeat;
+  background-size: contain;
+  background-position: center;
+}
+
+.test-drive__link {
+  display: flex;
+  width: 89px;
+  height: 18px;
+  font-size: 13px;
+  line-height: 16px;
+  font-style: normal;
+  font-weight: bold;
+  text-decoration: none;
+  color: #000;
+  margin: 0 40px 0 0;
+  text-transform: uppercase;
+}
+
+.test-drive__link:hover {
+  display: flex;
+  font-size: 13px;
+  line-height: 16px;
+  font-weight: bold;
+  text-align: center;
+  text-decoration: none;
+  color: #f00;
+  cursor: pointer;
+  padding: 0;
+  border: 0;
+}
+
+/* Кнопка Заказать */
+
+.button {
+  display: flex;
+  width: 160px;
+  height: 40px;
+  font-size: 13px;
+  line-height: 16px;
+  font-family: "Montserrat", sans-serif;
+  text-align: center;
+  font-weight: bold;
+  text-transform: uppercase;
+}
+
+.test-drive__button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 13px;
+  text-align: center;
+  font-weight: bold;
+  border: 2px solid #f00;
+  color: #f00;
+  background-color: #fff;
+}
+
+.test-drive__button:hover {
+  font-size: 14px;
+  color: #f00;
+  border: 3px solid #f00;
+  background-color: #fff;
+  cursor: pointer;
+  font-weight: bold;
+  padding: 0;
+}
+
+.test-drive__button:focus {
+  display: flex;
+  color: #f00;
+  border: 1px solid #f00;
+  background-color: #fff;
+  cursor: pointer;
+  font-weight: bold;
+  padding: 0;
+  outline: none;
+  box-shadow: 0 0 0 3px;
+}
+
+.test-drive__button:active {
+  background-color: #fff;
+  border-color: #f00;
+  color: #f00;
+  font-size: 12px;
+}
+
+/* Кнопка галка */
+
+.section__btn-pointer {
+  display: flex;
+  width: 34px;
+  height: 20px;
+  padding: 0;
+  margin: 37px auto 0 auto;
+  box-shadow: none;
+  background: url(https://i.ibb.co/GFX8Xs7/Vector-1.png) no-repeat;
+  background-size: contain;
+  position: center;
+  outline: none;
+  border: 1px solid white;
+}
+
+.section__btn-pointer:hover {
+  width: 40px;
+  height: 25px;
+  cursor: pointer;
+  padding: 0;
+  border: 1px solid white;
+  outline: none;
+}
+
+.section__btn-pointer:active {
+  width: 40px;
+  height: 25px;
+  outline: none;
+  margin: 32px auto 0;
+  box-shadow: 0 0 0 3px grey;
+}
+
+/* Секция 2 Секция 2 Секция 2 Секция 2 Секция 2 Секция 2 Секция 2 Секция 2 Секция  Секция 22 Секция 2 Секция 2 Секция 2 Секция 2 Секция 2 Секция 2 Секция 2 */ 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Hi-Tesla/pullRequest?prid=5710464)